tips/TIP-0049/tip-0049.md

```diff
@@ -0,0 +1,957 @@
+```
+
+The hash of the above protocol parameter example: `0x0bf2082ddeaa7e9c6b01bb05f0b18fc0bf93280a0f45d6b7b3e7c700afaa9adb`.
```

Comment:
You sure this is the correct hash yeah? I get the same bytes but different hash.

Reply:
You're right, I'm taking the wrong params for hash.

Reply:
Works now, thanks